qa-brain.planx-pla.net/portal/gitops.json

```diff
@@ -211,12 +211,28 @@
     "filters": {
       "tabs": [
         {
-          "title": "Case",
+          "title": "Study Selection",
+          "fields": [
+            "program
```

Comment:
```suggestion
            "program",
```

Reply:
removed `program` since Abby confirmed that will not work